frontend/src/App.js

```diff
@@ -1,12 +1,20 @@
-import { BrowserRouter as Router } from "react-router-dom";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import "./index.css";
 import Navbar from "./Components/Navbar";
 import MiniNav from "./Components/MiniNavbar";
+import Home from "./Pages/Home";
+import Login from "./Pages/Login";
```

Comment:
And also default exports are considered a kind of bad practice. But its completely upto you. Just a suggestion